interface/src/avatar/AvatarActionHold.cpp

```diff
@@ -119,8 +119,7 @@ void AvatarActionHold::doKinematicUpdate(float deltaTimeStep) {
         worldTrans.setRotation(glmToBullet(_rotationalTarget));
         rigidBody->setWorldTransform(worldTrans);
 
-        ownerEntity->setPosition(_positionalTarget);
-        ownerEntity->setRotation(_rotationalTarget);
+        motionState->dirtyInternalKinematicChanges();
```

Comment:
DO NOT poke the entity in the middle of the Action update (such code puts hurdles on our path toward the **Glorious Future** where the physics simulation is on its own thread).